repo/packages/E/etcd/0/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "etcd",
+  "version": "0.0.1",
+  "scm": "https://github.com/mesosphere/etcd-mesos",
+  "maintainer": "support@mesosphere.io",
+  "description": "A distributed consistent key-value store for shared configuration and service discovery.",
+  "framework": true,
+  "images": {
+    "icon-small": "https://downloads.mesosphere.io/etcd-mesos/assets/etcd-small.png",
+    "icon-medium": "https://downloads.mesosphere.io/etcd-mesos/assets/etcd-medium.png",
+    "icon-large": "https://downloads.mesosphere.io/etcd-mesos/assets/etcd-large.png"
+  },
+  "tags": ["mesosphere", "framework", "consensus"],
+  "preInstallNotes": "In order for etcd to start successfully all resources must be available in the cluster including ports, CPU shares and RAM.\nWe recommend a minimum of 3 nodes with 1 CPU share and 128 MB of RAM available for use by the etcd service.\nNote that the service is alpha and there may be bugs, including possible data loss, incomplete features, incorrect documentation or other discrepancies.",
+  "postInstallNotes": "Once the cluster initializes (<1 minute if offers are available), etcd proxies may connect by passing the argument -discovery-srv={{etcd.framework-name}}.mesos, and you may discover live members by querying SRV records for _etcd-server._tcp.{{etcd.framework-name}}.mesos"
```

Comment:
it's buried in dcos-cli somewhere, marathon.json and options.json are
munged together.

On Mon, Sep 28, 2015 at 9:19 PM, Tyler Neely <notifications@github.com>
wrote:

> In repo/packages/E/etcd/0/package.json
> <https://github.com/mesosphere/multiverse/pull/7#discussion_r40625346>:
>
> > @@ -0,0 +1,16 @@
> > +{
> > +  "name": "etcd",
> > +  "version": "0.0.1",
> > +  "scm": "https://github.com/mesosphere/etcd-mesos",
> > +  "maintainer": "support@mesosphere.io",
> > +  "description": "A distributed consistent key-value store for shared configuration and service discovery.",
> > +  "framework": true,
> > +  "images": {
> > +    "icon-small": "https://downloads.mesosphere.io/etcd-mesos/assets/etcd-small.png",
> > +    "icon-medium": "https://downloads.mesosphere.io/etcd-mesos/assets/etcd-medium.png",
> > +    "icon-large": "https://downloads.mesosphere.io/etcd-mesos/assets/etcd-large.png"
> > +  },
> > +  "tags": ["mesosphere", "framework", "consensus"],
> > +  "preInstallNotes": "In order for etcd to start successfully all resources must be available in the cluster including ports, CPU shares and RAM.\nWe recommend a minimum of 3 nodes with 1 CPU share and 128 MB of RAM available for use by the etcd service.\nNote that the service is alpha and there may be bugs, including possible data loss, incomplete features, incorrect documentation or other discrepancies.",
> > +  "postInstallNotes": "Once the cluster initializes (<1 minute if offers are available), etcd proxies may connect by passing the argument -discovery-srv={{etcd.framework-name}}.mesos, and you may discover live members by querying SRV records for _etcd-server._tcp.{{etcd.framework-name}}.mesos"
>
> Is it legit for me to use moustache templating in package.json? I searched
> a few github repos but I have no idea where this logic lives x_x
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/multiverse/pull/7/files#r40625346>.
>